config/routes.rb

```diff
@@ -37,6 +37,9 @@
           member do
             post 'sync', to: 'studies#sync_study'
           end
+          member do
```

Comment:
This doesn't need it's own `member` block, you can add it to the above one:

```
member do
  post 'sync', to: 'studies#sync_study'
  get 'manifest', to: 'studies#generate_manifest'
end
````